vector/src/main/res/layout/activity_emoji_reaction_picker.xml

```diff
@@ -39,8 +39,9 @@
         <com.google.android.material.tabs.TabLayout
             android:id="@+id/tabs"
             android:layout_width="match_parent"
-            android:layout_height="40dp" />
+            android:layout_height="40dp"
+            app:tabMode="auto" />
```

Comment:
With this fix, on big screen the user has to scroll.

I prefer to fix the issue by adding:

```
app:tabPaddingEnd="0dp"
app:tabPaddingStart="0dp"
```

instead of tabMode auto

Reply:
Your fix looks fine on my devices (one 4", one 5.5" screen) even for the largest proposed display settings, so sounds like a good idea :+1:

Reply:
Ok, thanks for testing. Will you update the PR then?

Reply:
Pushed!

Reply:
thanks